projects/packages/forms/src/dashboard/inbox/index.js

```diff
@@ -95,34 +15,7 @@ const Inbox = () => {
 		navigate( '/landing' );
 	}, [ navigate ] );
 
-	const onTabSelect = useCallback(
-		newStatusValue => {
-			setSearchParams( previousSearchParams => {
-				const _searchParams = new URLSearchParams( previousSearchParams );
-				_searchParams.set( 'status', newStatusValue );
-				return _searchParams;
-			} );
-		},
-		[ setSearchParams ]
-	);
-	return (
-		<>
-			<div className="jp-forms__layout-header">
-				<HStack justify="flex-end">
-					<ExportResponsesButton />
-				</HStack>
-			</div>
-			<TabPanel
-				className="jp-forms__inbox-tabs"
-				activeClass="active-tab"
-				initialTabName={ [ 'inbox', 'spam', 'trash' ].includes( urlStatus ) ? urlStatus : 'inbox' }
-				onSelect={ onTabSelect }
-				tabs={ tabs }
-			>
-				{ () => <InboxView /> }
-			</TabPanel>
```

Comment:
There are a lot of code removed from this file. But the root change is that we are removing the TabPanel. From there, all other code removed was supporting that component, like the especially the useTabItems and onTabSelect methods. Very similar logic now lives in the InboxStatusToggle component.